### PR TITLE
Update changelog; provide notes on F# analysis

### PR DESCRIPTION
**Pull request by StachuDotNet**

### CODE REVIEW

**Comment by StachuDotNet** [comment]
I'm not very confident in the wording here. Would appreciate some feedback @pbiggar

**Comment by pbiggar** [comment]
I see what you're going for here. Let me take a stab at it in a different PR, I think this is a place where showing is better than telling.

File: docs/reference/changelog.md [hunk 1]
```diff
@@ -4,6 +4,30 @@ title: Changelog
...
+OCaml one.
+
+If you face any issues in in-Editor code execution,
+
+- please [let us know](https://github.com/darklang/dark/issues) so we can
+  address the issue.
+- you can locally revert to use the legacy OCaml execution by changing the URL
+  in your browser: if you add a `wasm-use-ocaml=true` query parameter, the old
+  WASM-compiled Analysis engine will be used instead. If you are using the
+  desktop Dark application, there is no way to switch back at this point.
```
[the remark this answers is not in the document]
Comment (pbiggar) [reply docs/reference/changelog.md:25]: eg `window.DarkAnalysis.useFSharp = true`

[the remark this answers is not in the document]
Comment (StachuDotNet) [reply docs/reference/changelog.md:25]: Good point!
`window.DarkAnalysis.wasmUseOcaml = true` would do the job.

[the remark this answers is not in the document]
Comment (StachuDotNet) [reply docs/reference/changelog.md:25]: though thinking now 'wasm' may be redundant - can switch to `useOcaml` if appropriate.

[the remark this answers is not in the document]
Comment (pbiggar) [reply docs/reference/changelog.md:25]: Yeah, that's a bit better. Same for the queryParam, maybe `use-ocaml-analysis=true`?

**Comment by pbiggar** [comment]
> I see what you're going for here. Let me take a stab at it in a different PR, I think this is a place where showing is better than telling.

See #227

**Comment by StachuDotNet** [comment]
Closing in favor of #227

**Pull request closed without merging**